docs/application/native/guides/messaging/push-server.md

```diff
@@ -52,30 +52,29 @@ To send push notifications:
 
      ```
      {
-         "messages":
-         [{
-             /* Other content */
-             "message": "action=ALERT&badgeOption=INCREASE&badgeNumber=1&alertMessage=Hi",
-             "appData": "{id:asdf&passwd:1234}",
-             /* Other content */
-         ]}
+         /* Other content */
+         "message": "action=ALERT&badgeOption=INCREASE&badgeNumber=1&alertMessage=Hi",
+         "appData": "{id:asdf&passwd:1234}",
+         /* Other content */
      }
      ```
 
-   - If you have data to send to the application but no need to notify the user, use the action field on the same level as the messages field, instead of within the messages field, and do not include the message field itself. In this case, the notification is delivered with the best effort.
+   - If you have to send data to the application, and if the user need not be notified immediately, use the action field at the same level as the message field, and do not include the message field. In this case, the notification is delivered with the best effort.
```

Comment:
The updated suggestion for this line is below.

If you have to send data to an application without notifying the user immediately, use the `action` field. The action field must be used at the same level as `message` field, and do not include the message field itself. In this case, the notification is delivered on best-effort basis.

@titicoco, however you applied the 1st suggestion at this line. Would you please update the PR again?

Reply:
@mijin85cho, updated as guided. Thank you.